pkg/deployments/README.md

```diff
@@ -117,6 +117,7 @@ Go to each deprecated deployment's readme file to learn more about why it is dep
 
 | Description                                      | Task ID                                                                                             |
 | ------------------------------------------------ | --------------------------------------------------------------------------------------------------- |
+| Weighted Pools of up to 8 tokens                 | [`20210418-weighted-pool`](./tasks/20210418-weighted-pool)                                          |
```

Comment:
It seems you forgot to delete the old task from the `tasks` directory? Also the link should point to the `deprecated` dir